### ISSUE TITLE
迁移训练YOLOv3 ResNet50_vd DCN DropBlock IoULoss模型遇到问题

### ISSUE THREAD

**Issue by HaiLongEmb** [PaddlePaddle/PaddleDetection#468]
配置信息如下：
metric: VOC
pretrain_weights: https://paddlemodels.bj.bcebos.com/object_detection/yolov3_r50vd_dcn_obj365_dropblock_iouloss.tar
#pretrain_weights: https://paddlemodels.bj.bcebos.com/object_detection/ResNet50_vd_dcn_db_obj365_pretrained.tar
weights: output_test_iouloss/yolov3_r50vd_dcn_db_iouloss_obj365_pretrained/model_final
num_classes: 3
finetune_exclude_pretrained_params: ['yolo_output']
use_fine_grained_loss: True

但运行提示:
----------------------
Error Message Summary:
----------------------
Error: The input's size along the split dimension must be evenly divisible by Attr(num_or_sections). But received Attr(num_or_sections) = 8, input(X)'s shape = [4, 50, 4], Attr(dim) = 0.
  [Hint: Expected input_axis_dim % num == 0, but received input_axis_dim % num:4 != 0:0.] at (D:\1.7.0\paddle\paddle/fluid/operators/split_op.h:41)
  [operator < split > error]

为什么输入的尺寸会有区别？

这个问题我找到了，YOLOv3Loss中的batch_size设置没有与TrainReader.batch_size一致

**Comment by HaiLongEmb** [comment]
我添加了一层忽略，ignore_params = ['yolo_output', 'create_parameter']才能正常训练，请问create_parameter这是什么层

**Comment by FDInSky** [comment]
create_parameter，这个创建参数变量，不是层，比如卷积的weight，bias

**Comment by HaiLongEmb** [comment]
> create_parameter，这个创建参数变量，不是层，比如卷积的weight，bias

有一点没理解，如果是创建参数变量，不应该改成不匹配的问题啊，我只是更改了最后输出的类别，会对这个有影响吗？因为我迁移训练YOLOv3 ResNet50_vd DCN的时候出现create_parameter不匹配的问题，是不是因为添加了use_fine_grained_loss的原因。

**Comment by HaiLongEmb** [comment]
发现问题了，只要是
YOLOv3Loss:中的batch_size改变默认值create_parameter就会出现不匹配。
